vision.org

```diff
@@ -0,0 +1,32 @@
+#+TITLE: The Future of the ediTDor
+#+AUTHOR: Pedram Hadjian
+
+* Vision
+
+The Web of Things enables us to represent operational digital twins in an interoperable way. The main problem for IoT developers is the access to and understanding of data. Data in the context of IoT can broadly be split into two different categories:
+
+1. Timeseries Data: data coming from sensors, oftentimes called signals, runtime data, samples or others
+
+2. Contextual Data: this type of data answers questions about types of devices, interpretation of timeseries values, topology of assets and locations, taxonomies and more. Sometimes this type of data is called the semantics.
+
+We do not have a consistent way to work with these kinds of data. While normal software developers happily whip out VSCode, IntelliJ, emacs or vim, install a language server and make use of a myriad of libraries to get things done, IoT developers struggle to even access and understand data, let alone deliver value on top of it.
+
+In this open source project, we set out to change this. The vision is to bridge the gap between the two scenarios. We will focus on the developer experience and if we succeed, an IoT developer will be able to create or better just fetch device twins, model the context, write code and deploy to production.
+
+The vision is grand, but we need to start small. At Siemens we published the open source [[https://github.com/eclipse/editdor][ediTDor]] project, which helps to write and syntactically validate TMs/TDs. We will pick this up and drive it towards an awesome Digital Twin development environment.
+
+* Target Audience
+
+/IoT Developer/ is a very abstract role and for many an over-simplification. There are developers who create protocol driver code, experts who are knowledgeable about a domain, ontologists, application developers, system integrators etc.
```

Comment:
... OT developers.

Reply:
https://github.com/web-of-things-open-source/proposal/commit/1abdb344aad1f547f36bbcf116c1466007d25a92
- Added "OT developers" as a role